server/utils/applications/getApplicationNoticeType.test.ts

```diff
@@ -0,0 +1,40 @@
+import { add } from 'date-fns'
+
+import { DateFormats } from '../dateUtils'
+import { getArrivalDate } from './utils'
+import { getApplicationNoticeType } from './getApplicationNoticeType'
+
+import applicationFactory from '../../testutils/factories/application'
+
+jest.mock('./utils')
+
+describe('getApplicationNoticeType', () => {
+  const application = applicationFactory.build({})
+
+  it('returns emergency if the arrival date is less than seven days away', () => {
+    const date = add(new Date(), { days: 4 })
+    const arrivalDate = DateFormats.dateObjToIsoDate(date)
```

Comment:
Rather than mock `getArrivalDate` could we use `updateApplicationReleaseDate` to add the date?

Reply:
I think I prefer our unit tests to be as [solitary](https://martinfowler.com/bliki/UnitTest.html) as possible, that way if the implementation of `getArrivalDate` changes, we don't have to update code in a ton of other places.